controllers/windowsmachine_controller.go

```diff
@@ -519,3 +512,18 @@ func (r *WindowsMachineReconciler) isWindowsMachineHealthy(machine *mapi.Machine
 
 	return true
 }
+
+// getInternalIPAddress returns the internal IP address of the Machine
+func getInternalIPAddress(addresses []core.NodeAddress) (string, error) {
+	// Get the IP address associated with the Windows machine, if not error out to requeue again
+	if len(addresses) == 0 {
+		return "", errors.New("no IP addresses defined")
+	}
+	for _, address := range addresses {
+		// Only return the IPv4 address
```

Comment:
Is it possible for a cluster to only have ipv6 addresses?

Reply:
It is possible. But addressing that is out of the scope of this PR. We might need to holistically look at IPv6.